backend/core/src/shared/units-transformations.ts

```diff
@@ -100,16 +106,20 @@ const hmiData = (units: Units, maxHouseholdSize: number) => {
 
   const getYearlyRangeValue = (incomeRange: MinMaxCurrency) => {
     return incomeRange.min === incomeRange.max
-      ? incomeRange.min
-      : `${incomeRange.min} - ${incomeRange.max}`
+      ? `listings.annualIncome*income:${incomeRange.min}`
+      : `listings.annualIncomeRange*from:${incomeRange.min}*to:${incomeRange.max}`
+  }
+
+  const yearlyCurrencyStringToMonthly = (minimum: string) => {
```

Comment:
This argument is used for min and max below, not sure, but maybe we should rename it? :)

Reply:
good call!